debian/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM debian
+
+RUN apt-get update && apt-get install -y curl python2.7 apt-transport-https lsb-release
+RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
+    echo "deb https://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+ENV HOME /
+RUN  apt-get update && apt-get install -y google-cloud-sdk
```

Comment:
- extra space before `apt-get update`
- you can use `apt-get -qqy update && apt-get -qqy install` for less verbosity